state/AuthFunctions.ts

```diff
@@ -32,8 +38,6 @@ export const useAuthFunctions = () => {
   const attemptLogout = async () => {
     await logout()
     localStorage.clear()
-    localStorage.removeItem('SavingSatoshiProgress')
-    setCourseProgress(defaultProgressState)
```

Comment:
Are we sure we should remove these lines? When I log out, a bunch of challenges remain unlocked.

Reply:
I think we can keep `setCourseProgress(defaultProgessState)` if we also redirect the user to the `chaptersUrl` with no need to save the local Progress if they have an account.
But localStorage.clear() "should" do what we need already